packages/aws-cdk-lib/aws-rds/lib/cluster-engine.ts

```diff
@@ -806,15 +840,30 @@ export class AuroraPostgresEngineVersion {
    * @deprecated Version 11.15 is no longer supported by Amazon RDS.
    */
   public static readonly VER_11_15 = AuroraPostgresEngineVersion.of('11.15', '11', { s3Import: true, s3Export: true });
-  /** Version "11.16". */
+  /**
+   * Version "11.16"
+   * @deprecated Version 11.16 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_11_16 = AuroraPostgresEngineVersion.of('11.16', '11', { s3Import: true, s3Export: true });
```

Comment:
[Docs](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraPostgreSQLReleaseNotes/AuroraPostgreSQL.Updates.html) mark Aurora PostgreSQL v11 deprecated up to 11.15.
Did you check a different source?

Reply:
The docs would also be out of date then, I'm using the SDK for reference. In this case, it's not even listed. Here is the CLI command:

```sh
$ aws rds describe-db-engine-versions --engine aurora-postgresql --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion=="11.16")'
```